### ISSUE TITLE
[translations] Option to skip & rollback if error occurs

### ISSUE THREAD

**Issue by DutchmanNL** [AlCalzone/release-script#126]
It would be benefit full if you can chose that the release script should update the ip-package.json or only the version.
Also, like currently, if an error occurs on API level the script should rollback any modifications and/or allow to continue ad ignore the error

**Comment by DutchmanNL** [comment]
> Heh, I really need to finish #118

In Addition i still like the Option to skip Translation… like this Example 😂

![6B9BD0ED-555F-4C6B-82E7-322259298784](https://user-images.githubusercontent.com/7318445/173050367-0e380dd4-2049-44c3-a2de-82ae6ecf3fe8.jpeg)

**Comment by mcm1957** [comment]
Some comment based on my experience

Skipping translations which are already present would be a good feature if requested. Maybe the skipping should be limited to specified languages as most of us will not be able to update or check russion, chinese, ...

Personally I would be satisfied if I could verify the generated (german) text and adapt the english original with very little effort. Currently I must abort the release run, revert the git changes and then change the english text. Maybe a "retranslate" loop would be useful. Or the option to cancel the releasescript and revert the changes already done.

**Comment by mcm1957** [comment]
see slightly related issue #125

**Comment by DutchmanNL** [comment]
review of translations is implemented

**Comment by AlCalzone** [comment]
I don't think this issue is done? This is about rollback in case of an error.

**Comment by DutchmanNL** [comment]
> I don't think this issue is done? This is about rollback in case of an error.

Good point, would be even more additional value if indeed rollback would also be possible

**Comment by AlCalzone** [comment]
Rollback is fixed by https://github.com/AlCalzone/release-script/pull/205
